### PR TITLE
Update computed_metric-elastic.search.json

### PR DESCRIPTION
**Pull request by CryptoPishka**
Adding new metric netuitive.linux.elasticsearch.jvm.gccollectionpersecond.

### CODE REVIEW

**Comment by CryptoPishka** [comment]
Not sure why Travis is complaining. Json format looks good

File: analyticConfigurations/computed_metric-elastic.search.json [hunk 1]
```diff
@@ -300,6 +300,16 @@
           ],
           "fqn": "netuitive.linux.elasticsearch.indices.${1}.warmer.avg_per_sec"
         }
+      },
+      {
+        "match": "netuitive.linux.elasticsearch.jvm.gccollectionpersecond",
```
Comment (rclagett) [inline analyticConfigurations/computed_metric-elastic.search.json]: Could we make "collection" plural?
```suggestion
        "match": "netuitive.linux.elasticsearch.jvm.gccollectionspersecond",
```

File: analyticConfigurations/computed_metric-elastic.search.json [hunk 2]
```diff
@@ -300,6 +300,16 @@
...
+        "properties": {
+          "expressions": [
+            "${elasticsearch.jvm.gc.collection.count}.actual / 300.0"
+          ],
+          "fqn": "netuitive.linux.elasticsearch.jvm.gccollectionpersecond",
```
Comment (rclagett) [inline analyticConfigurations/computed_metric-elastic.search.json]:
```suggestion
          "fqn": "netuitive.linux.elasticsearch.jvm.gccollectionspersecond",
```

**Review by rclagett — CHANGES_REQUESTED** [review]
Thanks for the updates! Sorry, I missed this in the first review, could you also add an entry to the HISTORY file stating these changes?

**Review by rclagett — CHANGES_REQUESTED** [review]
We don't bump version on dev, we do it on master.

Could you remove v1.10.0 and place your entry under Version next?

**Review by rclagett — APPROVED** [review]
Looks great thank you!

**Pull request merged**